build issue

@hoganwr, @matentzn, I'm hoping you can help with a build problem. There seems to be an issue with owl remove,  in particular with the building of mono-import.owl.  I get this error:

INVALID FORMAT ERROR unknown format: owlremove

I tried running with the -vvv flag to see the whole dump. The tail end is below. If you can share any insight, I'd be most grateful.

thanks..

25-04-17 17:02:24,956 WARN  org.obolibrary.robot.OntologyHelper - <http://purl.obolibrary.org/obo/IAO_0000600> does not exist.
2025-04-17 17:02:24,956 WARN  org.obolibrary.robot.OntologyHelper - <http://purl.obolibrary.org/obo/APOLLO_SV_00000397> does not exist.
2025-04-17 17:02:24,956 WARN  org.obolibrary.robot.OntologyHelper - <http://purl.obolibrary.org/obo/APOLLO_SV_00000595> does not exist.
2025-04-17 17:02:24,956 WARN  org.obolibrary.robot.OntologyHelper - <http://purl.obolibrary.org/obo/IAO_0000118> does not exist.
2025-04-17 17:02:24,957 DEBUG org.semanticweb.owlapi.utilities.Injector - Injecting object uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl@a6efbd8
2025-04-17 17:02:24,957 DEBUG org.semanticweb.owlapi.utilities.Injector - Injecting values [[org.semanticweb.owlapi.rio.RioTrigParserFactory@1cbf0a1e, org.semanticweb.owlapi.rio.RioTurtleParserFactory@57b93578, org.semanticweb.owlapi.oboformat.OBOFormatOWLAPIParserFactory@880be43, org.semanticweb.owlapi.rio.RioBinaryRdfParserFactory@765355b0, org.semanticweb.owlapi.owlxml.parser.OWLXMLParserFactory@468b0aa3, org.semanticweb.owlapi.rio.RioTrixParserFactory@772626ce, org.semanticweb.owlapi.rdf.rdfxml.parser.RDFXMLParserFactory@677431f9, org.semanticweb.owlapi.functional.parser.OWLFunctionalSyntaxOWLParserFactory@5914e724, org.semanticweb.owlapi.rio.RioNTriplesParserFactory@3b9a1d0a, org.semanticweb.owlapi.rio.RioRDFXMLParserFactory@489fb12d, org.semanticweb.owlapi.krss2.parser.KRSS2OWLParserFactory@f35dd66, org.semanticweb.owlapi.manchestersyntax.parser.ManchesterOWLSyntaxOntologyParserFactory@2c767ec3, org.semanticweb.owlapi.rdf.turtle.parser.TurtleOntologyParserFactory@2459784c, org.semanticweb.owlapi.rio.RioNQuadsParserFactory@4f4366f6, org.semanticweb.owlapi.rio.RioJsonParserFactory@2eae98d9, org.semanticweb.owlapi.rio.RioJsonLDParserFactory@689387cb, org.semanticweb.owlapi.rio.RioN3ParserFactory@23d149e9, org.semanticweb.owlapi.rio.RioRDFaParserFactory@61a822f0]] on method public void uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.setOntologyParsers(java.util.Set).
2025-04-17 17:02:24,957 DEBUG org.semanticweb.owlapi.utilities.Injector - Injecting values [[uk.ac.manchester.cs.owl.owlapi.OWLOntologyFactoryImpl@629af07b]] on method public void uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.setOntologyFactories(java.util.Set).
2025-04-17 17:02:24,957 DEBUG org.semanticweb.owlapi.utilities.Injector - Injecting values [[org.semanticweb.owlapi.rio.RioN3StorerFactory@12a4a404, org.semanticweb.owlapi.krss2.renderer.KRSS2OWLSyntaxStorerFactory@62b1b1b0, org.semanticweb.owlapi.latex.renderer.LatexStorerFactory@1bb8895a, org.semanticweb.owlapi.dlsyntax.renderer.DLSyntaxStorerFactory@24881fec, org.semanticweb.owlapi.rio.RioNQuadsStorerFactory@1d7989bf, org.semanticweb.owlapi.rio.RioJsonStorerFactory@2fb94f5e, org.semanticweb.owlapi.rio.RioRDFXMLStorerFactory@6199a750, org.semanticweb.owlapi.rio.RioTrigStorerFactory@56aaabe3, org.semanticweb.owlapi.dlsyntax.renderer.DLSyntaxHTMLStorerFactory@4bdb1b16, org.semanticweb.owlapi.rio.RioTurtleStorerFactory@7d53f9dc, org.semanticweb.owlapi.owlxml.renderer.OWLXMLStorerFactory@4047af96, org.semanticweb.owlapi.rio.RioJsonLDStorerFactory@4baa4eb8, org.semanticweb.owlapi.oboformat.OBOFormatStorerFactory@71756e3, org.semanticweb.owlapi.rio.RioTrixStorerFactory@7cff4f8, org.semanticweb.owlapi.functional.renderer.FunctionalSyntaxStorerFactory@5ecb4013, org.semanticweb.owlapi.rdf.turtle.renderer.TurtleStorerFactory@1ff7896f, org.semanticweb.owlapi.rio.RioBinaryRdfStorerFactory@583afd02, org.semanticweb.owlapi.rio.RioNTriplesStorerFactory@4bfcb26, org.semanticweb.owlapi.rdf.rdfxml.renderer.RDFXMLStorerFactory@611e9cc, org.semanticweb.owlapi.manchestersyntax.renderer.ManchesterSyntaxStorerFactory@6b676a99]] on method public void uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.setOntologyStorers(java.util.Set).
2025-04-17 17:02:24,957 DEBUG org.semanticweb.owlapi.utilities.Injector - Loading file META-INF/services/org.semanticweb.owlapi.model.OWLOntologyIRIMapper
2025-04-17 17:02:24,957 DEBUG org.semanticweb.owlapi.utilities.Injector - No files found for META-INF/services/org.semanticweb.owlapi.model.OWLOntologyIRIMapper
2025-04-17 17:02:24,957 DEBUG org.semanticweb.owlapi.utilities.Injector - Injecting values [[]] on method public void uk.ac.manchester.cs.owl.owlapi.OWLOntologyManagerImpl.setIRIMappers(java.util.Set).
INVALID FORMAT ERROR unknown format: owlremove
For details see: http://robot.obolibrary.org/errors#invalid-format-error
java.lang.IllegalArgumentException: errors#INVALID FORMAT ERROR unknown format: owlremove
	at org.obolibrary.robot.CommandLineHelper.maybeSaveOutput(CommandLineHelper.java:671)
	at org.obolibrary.robot.RemoveCommand.execute(RemoveCommand.java:202)
	at org.obolibrary.robot.CommandManager.executeCommand(CommandManager.java:244)
	at org.obolibrary.robot.CommandManager.execute(CommandManager.java:188)
	at org.obolibrary.robot.CommandManager.main(CommandManager.java:135)
	at org.obolibrary.robot.CommandLineInterface.main(CommandLineInterface.java:61)
Caused by: java.lang.IllegalArgumentException: errors#INVALID FORMAT ERROR unknown format: owlremove
	at org.obolibrary.robot.IOHelper.getFormat(IOHelper.java:712)
	at org.obolibrary.robot.CommandLineHelper.maybeSaveOutput(CommandLineHelper.java:665)
	... 5 more
make: *** [imports/mondo_import.owl] Error 1

--- a/src/ontology/Makefile
+++ b/src/ontology/Makefile
@@ -73,10 +73,10 @@ $(MIRRORDIR)/mondo.owl: | build/robot.jar
   
 $(IMPORTDIR)/mondo_import.owl: $(MIRRORDIR)/mondo.owl $(IMPORTDIR)/seed.txt | build/robot.jar
 	$(ROBOT) \
-		extract -I $(OBO)/mondo/subsets/mondo.owl  -T $(IMPORTDIR)/seed.txt --copy-ontology-annotations true --force true --method subset \
-	remove --base-iri http://purl.obolibrary.org/obo/MONDO_ --axioms external --preserve-structure false --trim false --output $@\
-	remove $(patsubst %, --term %, $(ANNOTATION_PROPERTIES)) -T $(IMPORTDIR)/seed.txt --select complement \
-	annotate --ontology-iri "$(MIDAS)/$@" --output $@
+		extract -i $(MIRRORDIR)/mondo.owl  -T $(IMPORTDIR)/seed.txt --copy-ontology-annotations false --force true --method subset \
+		remove --base-iri http://purl.obolibrary.org/obo/MONDO_ --axioms external --preserve-structure false --trim false --output $@ \
+		remove $(patsubst %, --term %, $(ANNOTATION_PROPERTIES)) -T $(IMPORTDIR)/seed.txt --select complement \
+		annotate --ontology-iri "$(MIDAS)/$@" --output $@
 
 midas-data.owl: $(SRC) $(IMPORTS) | build/robot.jar
 	$(ROBOT) merge --input $< \

